inc/saiport.h

```diff
@@ -2284,6 +2296,27 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_OPER_PORT_FEC_MODE,
 
+    /**
+     * @brief Enable host tx signal
+     * will be used for independent module, to trigger
```

Comment:
Add empty linę before description and się capital letter, same for attribute below

Reply:
thanks for review, fix updated